src/Symfony/Component/Console/Tests/Fixtures/application_1.xml

```diff
@@ -24,34 +24,31 @@
         </argument>
       </arguments>
       <options>
-        <option name="--format" shortcut="" accept_value="1" is_value_required="1" is_multiple="0">
+        <option name="--format" shortcut="" accept_value="1" is_value_required="1" is_multiple="0" is_negatable="0">
           <description>The output format (txt, xml, json, or md)</description>
           <defaults>
             <default>txt</default>
           </defaults>
         </option>
-        <option name="--raw" shortcut="" accept_value="0" is_value_required="0" is_multiple="0">
+        <option name="--raw" shortcut="" accept_value="0" is_value_required="0" is_multiple="0" is_negatable="0">
           <description>To output raw command help</description>
         </option>
-        <option name="--help" shortcut="-h" accept_value="0" is_value_required="0" is_multiple="0">
+        <option name="--help" shortcut="-h" accept_value="0" is_value_required="0" is_multiple="0" is_negatable="0">
           <description>Display help for the given command. When no command is given display help for the &lt;info&gt;list&lt;/info&gt; command</description>
         </option>
-        <option name="--quiet" shortcut="-q" accept_value="0" is_value_required="0" is_multiple="0">
+        <option name="--quiet" shortcut="-q" accept_value="0" is_value_required="0" is_multiple="0" is_negatable="0">
           <description>Do not output any message</description>
         </option>
-        <option name="--verbose" shortcut="-v" shortcuts="-v|-vv|-vvv" accept_value="0" is_value_required="0" is_multiple="0">
+        <option name="--verbose" shortcut="-v" shortcuts="-v|-vv|-vvv" accept_value="0" is_value_required="0" is_multiple="0" is_negatable="0">
           <description>Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug</description>
         </option>
-        <option name="--version" shortcut="-V" accept_value="0" is_value_required="0" is_multiple="0">
+        <option name="--version" shortcut="-V" accept_value="0" is_value_required="0" is_multiple="0" is_negatable="0">
           <description>Display this application version</description>
         </option>
-        <option name="--ansi" shortcut="" accept_value="0" is_value_required="0" is_multiple="0">
-          <description>Force ANSI output</description>
+        <option name="--ansi" shortcut="" accept_value="0" is_value_required="0" is_multiple="0" is_negatable="1">
+          <description>Force/Disable ANSI output</description>
         </option>
-        <option name="--no-ansi" shortcut="" accept_value="0" is_value_required="0" is_multiple="0">
```

Comment:
Is it a BC break? Scripts that use the xml (or json) descriptor to provide auto-completion won't know that `--no-XXX` exists until they supports the `is_negatable` attribute.

Reply:
Works for me as-is.

Reply:
It IS a BC break and as IDEs are using these files, I think we need to keep the current format and remove the is_negatable attribute (same for all machine consumable formats). That way, there is no need for consumers to support the new options, and having two entries for negatable options make sense to me.

Reply:
restored. Please have a look to the `--no-XXX` description's wording